public/propTypes/analytics.js

```diff
@@ -0,0 +1,19 @@
+import { PropTypes } from 'react'
+
+export const pageCountStat = PropTypes.shape({
```

Comment:
Just to seperate this from the actual prop.pageCountStat, could we call this pageCountStatPropType or pageCountStatPropDef? I know it's long... but might be simpler when used.